lib/chef/mixin/windows_architecture_helper.rb

```diff
@@ -42,6 +36,16 @@ def wow64_architecture_override_required?(node, desired_architecture)
           is_i386_process_on_x86_64_windows?
       end
 
+      def forced_32bit_override_required?(node, desired_architecture)
+        desired_architecture == :i386 &&
+          node_windows_architecture(node) == :x86_64 &&
+          !is_i386_process_on_x86_64_windows?
+      end
+
+      def wow64_directory
+        Chef::ReservedNames::Win32::System.get_system_wow64_directory
+      end
+
       def with_os_architecture(node, architecture: nil)
```

Comment:
Good idea for a subsequent change, though I don't think anything we're doing here makes this worse than before the change.